Rules/CommonScripts/ItemLimits.as

```diff
@@ -0,0 +1,57 @@
+const string[] excludedFromTeamCheck = {"seed", "food", "fishy", "chicken", "shark", "bison", "greg", "log", "boulder", "tree_pine", "tree_bushy", "bush"};
+
+bool blobLimitExceeded(string blobName, CBlob@ caller = null) // returns true if exceeding the limit, false otherwise
+{	
+	string itemName = blobName;
+	if (blobName == "filled_bucket") itemName = "bucket";
```

Comment:
Filled_bucket is added as a shop item to builder shop in BuilderShop.as at line 44.
When builder shop receives command `this.getCommandID("shop made item")` it will check at line 122 if it is a "filled_bucket" and then create a "bucket" which is filled, and that is what should be counted in the item limit code.
It is because of this current implementation, that the item limit code has to check if "filled_bucket" is passed.